tests/test_packages/test_connections/test_p2p_libp2p_client/test_communication.py

```diff
@@ -31,7 +31,6 @@
 from aea.protocols.default.serialization import DefaultSerializer
 
 from ....conftest import (
-    MAX_FLAKY_RERUNS,
```

Comment:
Remove flaky mark, as communication tests should be 100% reliable with fix included in this pr